third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h

```diff
@@ -1935,8 +1966,90 @@ inline std::string StripTrailingSpaces(std::string str) {
 // standard functions as macros, the wrapper cannot have the same name
 // as the wrapped function.
 
+
+#if GTEST_OS_STARBOARD
+typedef int FILE;
+#endif
+
 namespace posix {
 
+#if GTEST_OS_STARBOARD
+
```

Comment:
This is exactly the type of stuff that i'd like to see completely eliminated with http://go/starboard-16-goals

No action needed here.

Reply:
b/288165638 - clean up bug.